subprojects/model-core/src/main/java/org/gradle/internal/reflect/validation/AbstractValidationProblemBuilder.java

```diff
@@ -86,6 +87,12 @@ public T addPossibleSolution(Supplier<String> solution, Action<? super SolutionB
         return Cast.uncheckedCast(this);
     }
 
+    @Override
+    public T isCachabilityProblemOnly() {
```

Comment:
```suggestion
    public T onlyAffectsCacheableWork() {
```

This method sounds very much like a getter, though it is something different. I think `cacheabilityProblemOnly()` or `onlyAffectsCacheableWork()` would be better. If you want to keep this method, it is missing an `e`.

Reply:
I had the same comment to myself after using it in the next PR :D